components/01-visual-styling/02-typography/09-table/table.scss

```diff
@@ -1,34 +1,58 @@
 /* components/01-visual-styling/02-typography/09-table/table.scss */
 /* BEGIN table.scss */
+table {
+	width: 100%;
+	padding: 0px 15px;
+	margin-bottom: 20px;
+}
 
-.table-design tr th {
-	padding: 12px 25px;
+th{
+	font-weight: bold !important;
+	color: $orange-a11y;
 }
-.table-design tr td {
+
+th, td {border: 1px solid $blue; padding:12px; vertical-align:top;}
+
+tr th {
 	padding: 12px 25px;
 }
-.table-design tbody>tr:nth-child(2n) {
-	background-color: $white-b;
-	border-top: 1px solid $grey;
-	border-bottom: 1px solid $grey;
-}
-.table-design {
-	width: 100%;
-	padding: 0px 15px;
-}
-.table-design thead {
-	background-color: $orange-a11y;
+
+tr td {
+	padding: 12px 25px;
 }
-.table-design thead th {
+
+thead th {
 	font-size: 17px;
 	font-weight: 500;
-	color: $white;
 }
-.table-design tr td {
+
+tr td {
 	padding: 12px 25px;
 	font-size: 17px;
 	color: $blue;
 }
+
+.table-design thead th {
+	color: $white;
+}
+
+.table-design th, .table-design td {border: 0px;}
+
+
+
+.table-design tbody>tr {
+	border-bottom: 1px solid $grey;
+
+	&:nth-child(2n) {
```

Comment:
I think that there might not be enough contrast between the colored row and its text, accessability-wise?

Maybe make the text of the colored row white? (Example of CSS)
Screen Shot 2022-06-20 at 4 28 35 PM

Or make the colored row background lighter by using rgba? (Example of CSS)
Screen Shot 2022-06-20 at 4 32 34 PM

Reply:
I did check the contrast with the Wave tool, and it seemed to pass with the $grey-c, but not the $grey-d.